src/pages/app-development/starter-kit/project-setup.md

```diff
@@ -262,7 +262,7 @@ You must configure Commerce to communicate with your project. Configuration incl
 
 ### Subscribe to events in Adobe Commerce
 
-Use the `bin/magento events:subscribe` command to subscribe to events, as described in [Subscribe and register events](https://developer.adobe.com/commerce/extensibility/events/configure-commerce/#subscribe-and-register-events). The following table defines the events for each supported entity that you must subscribe to and lists the required fields, if applicable.
+Use the `bin/magento events:subscribe` command to subscribe to events, as described in [Subscribe and register events](../../events/configure-commerce.md#subscribe-and-register-events). The following table defines the events for each supported entity that you must subscribe to and lists the required fields, if applicable.
```

Comment:
```
Use the `bin/magento events:subscribe` command to subscribe to events, as described in [Subscribe and register events](../../configure-commerce.md#subscribe-and-register-events). The following table defines the events for each supported entity that you must subscribe to and lists the required fields, if applicable.
```

Reply:
remove the `events/`

Reply:
or `[Subscribe and register events](../events/configure-commerce.md#subscribe-and-register-events)`